libs/platform/src/lib/components/thumbnail/thumbnail.component.ts

```diff
@@ -29,6 +32,12 @@ export class ThumbnailClickedEvent<T extends ThumbnailComponent = ThumbnailCompo
 })
 export class ThumbnailComponent extends BaseComponent implements OnInit {
 
+    constructor(protected _changeDetectorRef: ChangeDetectorRef,
```

Comment:
same as before on typedoc comments. Also constructor should be placed after the fields.

Reply:
modified and added typedocs